gax/src/main/java/com/google/api/gax/batching/v2/RequestBuilder.java

```diff
@@ -38,6 +40,7 @@
  * @param <ElementT> The type of each individual element to be batched.
  * @param <RequestT> The type of the request that will contain the accumulated elements.
  */
+@BetaApi("The surface for batching is not stable yet and may change in the future.")
```

Comment:
Please add `@InternalExtensionOnly("For google-cloud-java client use only")`  as well